rust/parquet/src/column/page.rs

```diff
@@ -217,6 +218,17 @@ pub trait PageWriter {
     fn close(&mut self) -> Result<()>;
 }
 
+/// API for iterating over pages of columns in a parquet file.
```

Comment:
Can we make it explicit that this is iterator over pages of a _specific_ column in the file? Also we need to improve the sentence "It's mainly used to decouple high level readers from low level file readers." - I got confused when reading it.

Reply:
Fixed.